src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -13,6 +13,11 @@
 import org.apache.spark.api.java.JavaSparkContext;
 import org.apache.spark.api.java.function.FlatMapFunction;
 import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.*;
```

Comment:
Yeah I know -- not sure why IntelliJ does this. I don't think it's worth fixing, since the next time the file is modified the imports will get auto-cleaned up anyway